keps/sig-node/127-user-namespaces/README.md

```diff
@@ -200,6 +201,65 @@ If true or not present, uses the host user namespace (as today)
 If false, a new userns is created for the pod.
 By default it is set to `true`.
 
+### CRI changes
+
+The following messages will be added:
+
+```
+// A IDMapping describes a ID mapping for the user namespace used for the
+// pod.
+message IDMapping {
+    // host_id is the id on the host.
+    uint32 host_id = 1;
+    // container_id is the id in the container.
+    uint32 container_id = 2;
+    // length is the size of the range to map.
+    uint32 length = 3;
```

Comment:
Should we call this size to be consistent with the OCI runtime spec?

Reply:
sure, that makes sense

@rata what do you think?

Reply:
Yes, fine by me!

Reply:
fixed in the last commit

Reply:
Updated

Reply:
I had to revert this change, now I remember why I used Length: https://github.com/kubernetes/kubernetes/issues/87336